Owin.Security.Providers/LinkedIn/Provider/LinkedInAuthenticatedContext.cs

```diff
@@ -69,12 +70,17 @@ public LinkedInAuthenticatedContext(IOwinContext context, JObject user, string a
         /// </summary>
         public string Name { get; private set; }
 
-        public string Link { get; private set; }
+        /// <summary>
+        /// Get the user's first name
+        /// </summary>
+        public string GivenName { get; private set; }
 
         /// <summary>
-        /// Gets the LinkedIn username
+        /// Get the user's last name
         /// </summary>
-        public string UserName { get; private set; }
```

Comment:
The library may not be using the `UserName` property internally anymore, but you have no guarantee that no one else is using it. 

We can go around in circles for days like this, but I am not going to accept a PR which has the potential of breaking other people's code. In the end I am the one who will get shout at when I release a minor version which does not have backwards compatibility, not you. 

So please put it back. Feel free however to mark it as `[Obsolete]` with a message as to the correct property to use.